packages/tuono/src/ssr/utils.ts

```diff
@@ -0,0 +1,36 @@
+function concatArrayBuffers(chunks: Array<Uint8Array>): Uint8Array {
+  const result = new Uint8Array(chunks.reduce((a, c) => a + c.length, 0))
+  let offset = 0
+  for (const chunk of chunks) {
+    result.set(chunk, offset)
+    offset += chunk.length
+  }
+  return result
+}
+
+async function streamToArrayBuffer(
+  stream: ReadableStream<Uint8Array>,
+): Promise<Uint8Array> {
+  const chunks: Array<Uint8Array> = []
+
+  // @ts-expect-error ReadableStream has AsyncIterator https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream#async_iteration
+  // eslint-disable-next-line @typescript-eslint/await-thenable
+  for await (const chunk of stream) {
+    chunks.push(chunk as Uint8Array)
+  }
```

Comment:
Probably we should update `tsconfig` `compilerOptions#lib`  to include `AsyncIterator` types